datastore/payload.go

```diff
@@ -20,17 +20,24 @@ type EventPayload struct {
 	Events   []interface{} `json:"events"`
 }
 
+type Event interface {
+	GetUID() string
+}
```

Comment:
I created this generic `Event` interface to make it a bit easier to DRY up the Prometheus exporter.